scripts/eateries/EateryList.js

```diff
@@ -1,5 +1,36 @@
-// imports useEat() and getEat() and EatHTML()
-import { getEatery, useEatery } from "./EateryProvider.js";
-// generates HTML list of all individual EatHTML() components
 
-// exports EatList()
\ No newline at end of file
+import { EateryHTML } from "./EateryHTMLGenerator.js";
+import { useEatery, getEatery } from "./EateryProvider.js";
+
+const contentTarget = document.querySelector(".container--eatery")
+const eventHub = document.querySelector(".main")
+
+eventHub.addEventListener("eaterySelected", event => {
+
+    const eateryThatWasSelected = event.detail.eateryId
+    const allEateries = useEatery()
+    const foundEatery = allEateries.find(eateryObj => {
+        return eateryObj.id === (parseInt(eateryThatWasSelected))
+    })
+    console.log(foundEatery)
+    render(foundEatery)
```

Comment:
Confirmed eatery dropdown renders, and the eatery selected populates within the itinerary preview after selecting an eatery. Great work!